app/controllers/internal/v0/notify_controller.rb

```diff
@@ -1,13 +1,25 @@
 module Internal
   module V0
     class NotifyController < ::ApplicationController
-      def notify
-        klass = params.require(:klass)
+      def notify_approval_request
         id = params.require(:id)
         payload = params.require(:payload)
 
         ActsAsTenant.without_tenant do
-          Catalog::Notify.new(klass, id, payload).process
+          Catalog::NotifyApprovalRequest.new(id, payload).process
+        end
+
+        json_response(nil)
```

Comment:
Maybe `head :no_content` instead? or are you specifically just wanting a 200 with no body?

Reply:
@syncrou Yeah, that was the original thinking.